npm2deb/__init__.py

```diff
@@ -457,7 +457,10 @@ def _get_Depends(self):
                     mapper.append_warning('error', dep, 'dependency %s '
                                           'not in debian' % (name))
                 version = dependencies[dep].replace('~', '')
+                version = dependencies[dep].replace('^', '')
                 if version:
+                    if version.find('.x') is not -1:
```

Comment:
I meant `if version.find('.x') is not -1` line,
Isn't `version = version.replace('.x', '.0')` sufficient without if block ?